Add initial version of 3scale alert SOPs included on 3scale v2.9 (#9)
3scale/3scale-Operations: sops/alerts/pod_crash_looping.adoc

:toc:
:toc-placement!:

= ThreescalePodCrashLooping

toc::[]

== Description

* This alert will trigger if a pod is on `CrashLoop` state because one of the containers crashes and is restarted indefinitely

== Troubleshooting

* Check pod logs/events to see the reason of the `CrashLoop`

> * Check if maybe container image registry is not reachable

Author: ```suggestion
```

> * Check if maybe there is an issue with the scheduled node (`OutofDisk`, `Insufficient CPU`, `MemoryPressure`, `Disk Pressure`...)

Author: ```suggestion
```

* Check if maybe container resources requests/limits are too low and OOMKiller is acting (container trying to allocate more memory than permitted)

> * Check if maybe deployment has incorrect parameters (like mounting not existing volumes, secrets, configmaps...)

Author: ```suggestion
```

:toc:
:toc-placement!:

= ThreescalePodCrashLooping

toc::[]

== Description

> * This alert will trigger if a pod is on `CrashLoop` state (being restarted regularly)

Reviewer: ```suggestion
* This alert will trigger if a pod is on `CrashLoop` state because one of the containers crashes and is restarted indefinitely
```

== Troubleshooting

* Check pod logs/events to see the reason of the `CrashLoop`

> * Check if maybe container image registry is not reachable

Reviewer: Are you sure that this one may trigger a crash loopback? Or the only the ThreescaleContainerWaiting alert?
